h/accounts/test/views_test.py

```diff
@@ -32,6 +32,8 @@ def configure(config):
     config.registry.registerUtility(SubmitForm, IProfileForm)
     config.registry.registerUtility(MagicMock(), IRegisterSchema)
     config.registry.registerUtility(MagicMock(), IRegisterForm)
+    config.registry.feature = MagicMock()
```

Comment:
Has it moved? This line on master is still accessing `registry.feature`: https://github.com/hypothesis/h/blob/master/h/accounts/views.py#L308